src/deployment/bitcoin_deployment.rs

```diff
@@ -0,0 +1,158 @@
+            },
+            script_sig: Script::new(),
+            sequence: 0xFFFFFFFD, // allow RBF
+            witness: Witness::new(),
+        };
+        transaction.input.push(input);
+        selected_utxos.push(utxo);
+    }
+
+    // Prepare Recipient output
+    let recipient_pub_key_hash = tx_spec
+        .recipient
+        .from_base58()
+        .expect("Unable to get bytes recipient btc address");
+    let txout = TxOut {
+        value: tx_spec.sats_amount,
+        script_pubkey: Builder::new()
+            .push_opcode(opcodes::all::OP_DUP)
+            .push_opcode(opcodes::all::OP_HASH160)
+            .push_slice(&recipient_pub_key_hash[1..21])
+            .push_opcode(opcodes::all::OP_EQUALVERIFY)
+            .push_opcode(opcodes::all::OP_CHECKSIG)
+            .into_script(),
+    };
+    transaction.output.push(txout);
+
+    // Prepare Sender change output
+    let sender_pub_key_hash = tx_spec
+        .expected_sender
+        .from_base58()
+        .expect("Unable to get bytes sender btc address");
+    let txout = TxOut {
+        value: cumulated_amount - tx_spec.sats_amount - tx_fee,
+        script_pubkey: Builder::new()
+            .push_opcode(opcodes::all::OP_DUP)
+            .push_opcode(opcodes::all::OP_HASH160)
+            .push_slice(&sender_pub_key_hash[1..21])
+            .push_opcode(opcodes::all::OP_EQUALVERIFY)
+            .push_opcode(opcodes::all::OP_CHECKSIG)
+            .into_script(),
+    };
+    transaction.output.push(txout);
+
+    (transaction, selected_utxos)
+}
+
+pub fn sign_transaction(
+    transaction: &mut Transaction,
+    utxos: Vec<ListUnspentResultEntry>,
+    signer: &SecretKey,
+) {
+    for (i, utxo) in utxos.into_iter().enumerate() {
+        let sig_hash_all = 0x01;
+        let script_pub_key = Script::from(utxo.script_pub_key.into_bytes());
+        let sig_hash = transaction.signature_hash(i, &script_pub_key, sig_hash_all);
+
+        let (sig_der, public_key) = {
+            let sig_hash_bytes = sig_hash.as_hash();
+            let message =
+                Message::from_slice(&sig_hash_bytes[..]).expect("Unable to create Message");
+            let secp = Secp256k1::new();
+            let signature = secp.sign_recoverable(&message, signer);
+            let public_key = PublicKey::from_secret_key(&secp, &signer);
+            let sig_der = signature.to_standard().serialize_der();
+            (sig_der, public_key)
+        };
+
+        transaction.input[i].script_sig = Builder::new()
+            .push_slice(&[&*sig_der, &[sig_hash_all as u8][..]].concat())
+            .push_slice(&public_key.serialize())
+            .into_script();
+    }
+}
+
+pub fn send_transaction_spec(
+    tx_spec: &BtcTransferSpecification,
+    signer: &SecretKey,
+) -> Result<bitcoincore_rpc::bitcoin::Txid, String> {
+    let auth = Auth::UserPass("devnet".to_string(), "devnet".to_string());
+    let bitcoin_node_rpc_url = "http://0.0.0.0:18443";
```

Comment:
Should these be pulled from Devnet.toml?

Reply:
Addressed in https://github.com/hirosystems/clarinet/pull/402/commits/ce821313ed66b9190826dd147cdcc477cf6908f0